api/web3server.go

```diff
@@ -565,7 +571,12 @@ func (svr *Web3Server) getProtocolVersion() (interface{}, error) {
 }
 
 func (svr *Web3Server) isSyncing() (interface{}, error) {
-	return false, nil
+	start, curr, highest := svr.coreService.SyningProgress()
+	return &syncingObject{
+		startingBlock: uint64ToHex(start),
+		currentBlock:  uint64ToHex(curr),
+		highestBlock:  uint64ToHex(highest),
+	}, nil
```

Comment:
should return `"result": false` if the chain is not syncing any more (reaches highest block)